### PR TITLE
Mobile home feature

### PR DESCRIPTION
**Pull request by shafiuyushawu**
Portfolio: finish mobile version

- Created Navbar for mobile responsiveness
- Created Profile for the portfolio

### CODE REVIEW

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @shafiuyushawu , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 
- [x] Does not have linter checks. please kindly check [here ](https://github.com/microverseinc/linters-config/tree/master/html-css)to add them
- [x] please kindly remove the microverse logo from your readme. you can keep the space empty if you don't yet have a personal logo to use
- [x] kindly include 1-3 potential features to be added in your future features section of readme.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -4,16 +4,69 @@
...
-            
-        </section>
+        <nav class="navbar">
+            <div class="nav__logo">
+                <a href="#">SHAFIU</a> 
+            </div>
+            <div class="nav__hamburger">
+                <img src="./images/hamburger.png" alt="not internet">
+            </div>
+        </nav>
```
Comment (Abenezer-Tilahun) [inline index.html]: - [ ] Kindly I recommend you to use `header` tag for toolbar section, Typically the `header `tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @shafiuyushawu   , 

Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻

✔️ No Linters error
✔️ You used correct Git Hub-Flow.
✔️ Professional  README
✔️ Good commit messages

almost

## Required Changes ♻️

_Check the comments below the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -4,16 +4,69 @@
...
     <div>
-        <header>
-
+        <header class="navbar">
+            <div class="nav__logo">
+                <a href="#">SHAFIU</a> 
+            </div>
+            <div class="nav__hamburger">
+                <img src="./images/hamburger.png" alt="not internet">
+            </div>
```
Comment (mckent05) [inline index.html]: - Good job done so far 👏 , you seem to have misunderstood the comment from the last reviewer, you were supposed to wrap the `nav` tag and its content in the `header` tag. Please fix this 🙏.
- Due to the changes you made from the last review, you seem to have introduced some bugs that changed the design of your site a little. Please ensure that your navbar looks the same way it was before the last review. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your nav bar before last review:
gqegqeg

Your navbar currently:
g3g

File: index.css [hunk 3]
```diff
@@ -0,0 +1,114 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap');
+
+body {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```
Comment (mckent05) [inline index.css:9]: **OPTIONAL** - Though your portfolio looks good 👌, it is very important that it fits well across all mobile screen sizes `<765px`. I suggest that you use [media queries](https://www.google.com/url?sa=t&source=web&rct=j&url=https://www.w3schools.com/css/css3_mediaqueries.asp&ved=2ahUKEwj8rfeh1sH8AhVO4qQKHQBJCsgQFnoECAcQAQ&usg=AOvVaw34me2GWSMjbN7a7mjFFhmF) to make your site responsive, please fix this 👍 👍

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @shafiuyushawu, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
almost there


 ## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Your portfolio is looking good 👌 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your code.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.css (hunk 3 above)
Comment (shafiuyushawu) [reply index.css:9]: @mckent05 
Thanks for the clarity

**Review by jerryowusu — APPROVED** [review]
Hi @shafiuyushawu  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

clapping-leonardo-dicaprio


## Highlights
All linters are passing :heavy_check_mark: 
Great effort in making the requested changes :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**